test/e2e/upgrade_test.go

```diff
@@ -100,35 +70,19 @@ func testTamperAnnotation(t *testing.T) {
 	testPrometheus(t)
 }
 
-// configureUpgradeTest carries out steps required before running tests for upgrade scenario.
-// The steps include -
-// 1. Scale down the operator to 0.
-// 2. Change Windows node version annotation to an invalid value
-// 3. Scale up the operator to 1
+// configureUpgradeTest carries out steps required before running tests for upgrade scenario. The steps include -
+// 1. Ensuring the correct amount of nodes have a proper annotation to start
+// 2. Ensuring the operator is running with 1 replica
 func (tc *testContext) configureUpgradeTest() error {
```

Comment:
Reviewers, as part of the individual test suites (network, upgrade, metrics, etc) within the e2e tests, which route should we take?
1. just assume that the cluster is in testable state (e.g. operator is up and, for this suite, correct # nodes initially have valid version annotation) -- this approach calls for the removal of this setup method
2. have guards to check, else fail fast/don't even run test case
3. try to coerce cluster to reach proper state